src/pages/docs/creators/javascript/guides/providers-filesystem.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Golem concepts: filesystem on remote node'
+description: Explore topics related to file system on a VM run on a remote node to better design your custom image.
+pageTitle: VM Filesystem - how the image design impacts my options.
+type: Guide
+---
+
+# Filesystem on VM run on a remote node.
+
+{% alert level="info" %}
+
+This example utilized the `golem/examples-outbound:latest` image deployed on one of the nodes in the `testnet`. Actual numbers will depend on the node on which you run such a task.
```

Comment:
What numbers? Numbers of providers with the image cached?

Reply:
If its just the number of providers with the image cached, then remove that part. Its not necessary